src/services/posts/record-likes.test.js

```diff
@@ -0,0 +1,27 @@
+const {expect} = require('chai')
+const db = require("../../db")
+const recordPostLikes = require("./record-likes")
+const {fixtures} = require('../../../test/utils')
+
+
+describe('RECORD LIKES on gif', () => {
+	let user
+	before(async ()=>{
+		 user = await fixtures.insertUser()  
+	})
```

Comment:
kindly add a test case for the custom error

Reply:
am i supposed to have a new describe and it block for this ?